service/src/main/java/uk/gov/mca/beacons/api/export/xlsx/backup/BackupToXlsxJobManager.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.mca.beacons.api.export.xlsx.backup;
+
+import java.nio.file.Path;
+import java.util.Date;
+import java.util.Objects;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.batch.core.BatchStatus;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.JobExecution;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersBuilder;
+import org.springframework.batch.core.launch.JobLauncher;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.mca.beacons.api.export.ExportFailedException;
+
+/**
+ * Initiates export jobs using Spring Batch.
+ */
+@Component
+@Slf4j
+public class BackupToXlsxJobManager {
+
+  private final JobLauncher jobLauncher;
+
+  // todo: how do I reuse one XlsxJobManager and decide at runtime which job I want to run
```

Comment:
Do you still need this comment?

Reply:
If poss I would like to know how to address the question in the comment but maybe we should save refactoring this feature for a future iteration